src/pages/events/createEvents/CreateEvent.tsx

```diff
@@ -0,0 +1,473 @@
+              <Paper key={item.id} className={classes.evaluatorCard}>
+                <Box className={classes.evaluatorCardHeader}>
+                  <Typography className={classes.evaluatorHeaderName}>{item.header}</Typography>
+                  <IconButton
+                    className={classes.deleteIconButton}
+                    onClick={() => {
+                      handleRemoveEvaluator(item)
+                    }}
+                    aria-label="delete"
+                  >
+                    <DeleteIcon />
+                  </IconButton>
+                </Box>
+                <Box className={classes.evaluatorCardInputBox}>
+                  <TextField
+                    className={classes.evaluateeCardInput}
+                    label={item.name}
+                    value={item.nameValue}
+                    variant="outlined"
+                    size="small"
+                    fullWidth
+                    onChange={(e) => onChangeEvaluatorName(item.id, e.target.value)}
+                  />
+                  <TextField
+                    className={classes.evaluateeCardInput}
+                    label={item.position}
+                    value={item.positionValue}
+                    variant="outlined"
+                    size="small"
+                    fullWidth
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                      onChangeEvaluatorPosition(item.id, e.target.value)
+                    }}
+                  />
+                  <TextField
+                    className={classes.evaluateeCardInput}
+                    label={item.email}
+                    value={item.emailValue}
+                    variant="outlined"
+                    size="small"
+                    autoComplete="email"
+                    fullWidth
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                      onChangeEvaluatorEmail(item.id, e.target.value)
+                    }}
+                  />
+                </Box>
+              </Paper>
+            ))}
+          </Grid>
+        </Box>
+        <Box>
+          <Box className={classes.eventInfoBoxEvaluator}>
+            <Typography className={classes.evaluatorText}>Evaluatees:</Typography>
+            <FormGroup className={classes.addButtonBox}>
+              <Box component="form" onSubmit={handleEvaluateesSubmit}>
+                <TextField
+                  InputLabelProps={{style: {fontSize: 14}}}
+                  className={`${classes.evaluatorInput} textField-remove-border`}
+                  label="Number of evaluatees"
+                  variant="outlined"
+                  error={evaluateeCountError}
+                  type="number"
+                  fullWidth
+                  size="small"
+                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                    setEvaluateesCount(parseInt(e.target.value, 10))
```

Comment:
Throw error when the number is with a point

Reply:
it's already throws error